docs/dev.md

```diff
@@ -153,7 +153,6 @@ $ yarn ios:run:emulator
 # Replace the host with your own host (find it with ifconfig)
 $ env DEV_HOST=192.168.1.36 yarn build:mobile:hot
 $ yarn ios:run # at this point the app is blank since it cannot access the files from your host
-$ env DEV_HOST=192.168.1.36 yarn watch:mobile:hot # launch a webpack-dev-server
```

Comment:
Instead of removing the line, I think it would be better to replace it by 

```
 $ env DEV_HOST=192.168.1.36 yarn watch:mobile # launch a webpack-dev-server
```

don't you think ?

Reply:
you're right, it's done